lib/views/teach_word_view.dart

```diff
@@ -112,23 +141,26 @@ class _TeachWordViewState extends State<TeachWordView>
               physics: const NeverScrollableScrollPhysics(),
               controller: _tabController,
               children: [
-                const TeachWordTabBarView(
+                TeachWordTabBarView(
+                    isBpmf: isBpmf,
                     sectionName: '看一看',
+                    word: word,
                     content: Image(
-                      height: 130,
-                      image: AssetImage(
-                          'lib/assets/img/evolution/' + demoChar + '.png'),
+                      width: 300,
+                      image: AssetImage('lib/assets/img/oldWords/$word.png'),
                     )),
                 TeachWordTabBarView(
+                    isBpmf: isBpmf,
+                    word: word,
                     sectionName: '聽一聽',
                     content: Column(
                       children: [
                         Container(
                           height: 300,
                           alignment: Alignment.center,
-                          child: const Text(demoChar,
+                          child: Text(word,
                               textAlign: TextAlign.center,
```

Comment:
Align to the 2-space indent style with previous codes. Remove additional indents from 4 to 2.

Reply:
umm... i think im not able to edit the format manually. the method i know is to close the "format on save", edit, save, and open it again.